packages/web/src/event.rs

```diff
@@ -398,24 +416,18 @@ impl HasFormData for WebFormData {
                 if let Ok(array) = value.dyn_into::<Array>() {
                     if let Some(name) = array.get(0).as_string() {
                         if let Ok(item_values) = array.get(1).dyn_into::<Array>() {
-                            let item_values: Vec<String> =
-                                item_values.iter().filter_map(|v| v.as_string()).collect();
-
-                            values.insert(name, FormValue::VecText(item_values));
+                            item_values
+                                .iter()
+                                .filter_map(|v| v.as_string())
+                                .for_each(|v| insert_value(&mut values, name.clone(), v));
                         } else if let Ok(item_value) = array.get(1).dyn_into::<JsValue>() {
-                            values.insert(name, FormValue::Text(item_value.as_string().unwrap()));
+                            insert_value(&mut values, name, item_value.as_string().unwrap());
                         }
                     }
                 }
             }
         }
 
-        // try to fill in select element values
```

Comment:
This branch makes on input for select elements work.

Reply:
In my case everything works without it

Reply:
Select multiple elements is broken. Try this example:

```rust
use dioxus::prelude::*;

fn main() {
    dioxus_web::launch(app);
}

fn app(cx: Scope) -> Element {
    let selected = use_state(cx,String::new);

    render! {
        select {
            multiple: true,
            oninput: |e| {
                selected.set(format!("{e:#?}"));
            },
            option { "Hello1" }
            option { "Hello2" }
            option { "Hello3" }
            option { "Hello4" }
        }

        pre { "{selected}" }
    }
}
```

https://github.com/DioxusLabs/dioxus/assets/66571940/5f8dd4ac-52e1-4d48-a477-f14e2e6283ca